code/game/objects/items/plushes.dm

```diff
@@ -736,20 +653,6 @@
 	icon_state = "scrubpuppy"
 	item_state = "scrubpuppy"
 
-/obj/item/toy/plush/borgplushie/seeking
-	icon_state = "seeking"
-	item_state = "seeking"
-
-/obj/item/toy/plush/borgplushie/neeb
-	icon_state = "neeb"
-	item_state = "neeb"
-
-/obj/item/toy/plush/borgplushie/bhijn
-	desc = "An adorable stuffed toy of a IPC."
-	icon_state = "bhijn"
-	item_state = "bhijn"
-	attack_verb = list("closed", "reworked", "merged")
-
 /obj/item/toy/plush/aiplush
```

Comment:
Pretty sure this is another player reference, but it can be moved with the rest later when a more iconically generic AI plushie is added.